src/plugins/controls/public/time_slider/components/time_slider.tsx

```diff
@@ -68,9 +67,6 @@ export const TimeSlider: FC<Props> = (props: Props) => {
       isOpen={isOpen}
       closePopover={() => timeSlider.dispatch.setIsOpen({ isOpen: false })}
       panelPaddingSize="s"
-      anchorPosition="downCenter"
-      disableFocusTrap
```

Comment:
Not sure why `disableFocusTrap` was here in the first place - it made the time slider completely inaccessible via keyboard controls.